src/Activation/ActivateExposureNotifications.tsx

```diff
@@ -17,7 +17,7 @@ import { isPlatformiOS } from "../utils"
 
 import { Spacing, Typography, Buttons, Colors } from "../styles"
 
-const ActivateProximityTracing: FunctionComponent = () => {
+const ActivateExposureNotifications: FunctionComponent = () => {
```

Comment:
Chatted, irl: The reasoning is that while `Proximity Tracing` may be better english, the whole point of this screen is to help users understand why the exposure notifications feature is not working. This is what it is called in the os, and it would be confusing to change the name in the app.